appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/AppyxInteractionsContainer.kt

```diff
@@ -50,6 +50,11 @@ import com.bumble.appyx.interactions.core.ui.property.motionPropertyRenderValue
 
 private val defaultExtraTouch = 48f.dp
 
+enum class GesturesRelation {
```

Comment:
1. I'd extract this to a standalone file in the gestures package
2. I'd rename to `GesturesReferencePoint`
3. I'd provide some comments on what they do

Reply:
Also for consistency, probably use singular rather than plural, the same as everywhere else